src/app/profile/payment-methods/bank-account/bank-account.component.js

```diff
@@ -0,0 +1,73 @@
+import angular from 'angular';
+import template from './bank-account.tpl';
+import displayAddressComponent from 'common/components/display-address/display-address.component';
+import editPaymentMethodModal from 'common/components/paymentMethods/editPaymentMethod/editPaymentMethod.modal.component';
+import deletePaymentMethodModal from 'common/components/paymentMethods/deletePaymentMethod/deletePaymentMethod.modal.component.js';
+import giveModalWindowTemplate from 'common/templates/giveModalWindow.tpl';
+import recurringGiftsComponent from '../recurring-gifts/recurring-gifts.component';
+
+
+class BankAccountController{
```

Comment:
Do you think this and the credit-card component could be combined? It looks like a lot of the same code. The template could show different content depending on the payment type.